src/contrib/mlir/runtime/cpu/cpu_callbacks.cpp

```diff
@@ -136,13 +136,12 @@ static mkldnn::memory convert_layout_if_diff(const mkldnn::memory::desc& lhs,
     }
 }
 
-static void convert_output_layout(const mkldnn::memory::desc& lhs,
+static void convert_output_layout(mkldnn::memory& reorder_in,
```

Comment:
We should use MLIR naming convention (camelCase) here except for public APIs. You could fix the ones you introduced in the PR, and the remaining in a separate PR.